lib/momento/generated/generate_protos.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -e
+set -x
+
+# Must specify the path to the proto files
+if [ -z "${PROTO_PATH}" ]; then
+  echo "PROTO_PATH is not set, please specify the path to the client-protos/proto directory"
+  exit 1
+fi
+
+# Remove existing generated code
+rm -f *_pb.rb
+
+# Copy files over temporarily
+cp ${PROTO_PATH}/*.proto ./
+
+# Remove files we don't want to generate code from
+rm -f ./httpcache.proto
+rm -f ./vectorindex.proto
+
+# Generate the code
+grpc_tools_ruby_protoc -I ./ --ruby_out=./ --grpc_out=./ ./*.proto
+
+# Remove the protos
+rm -f ./*.proto
+
+# In the *_pb.rb files, replace the require with require_relative
+sed -i '' -E "s/require '(.*)_pb'/require_relative '\1_pb'/g" *_pb.rb
+
+# Wrap the generated code in a module named MomentoProtos but only if not "module Scs"
```

Comment:
what does the `if not "module Scs"` part here mean?

Reply:
oops, stale comment from when I was trying to figure out how to wrap only the top-level module in MomentoProtos, will fix